wandb/sdk/wandb_run.py

```diff
@@ -1171,6 +1171,11 @@ def log_code(
                 files_added = True
                 art.add_file(file_path, name=save_name)
         if not files_added:
+            warnings.warn(
+                "No .py files or Dockerfiles detected in current directory. No code will be logged to your wandb run.",
```

Comment:
also the content of the warning seems too specific, let's maybe say something like (since they can modify the conditions of which files we log using the include_fn/exclude_fn:
 no relevant files were detected in the specified directory. No code will be ...